src/utils_social.js

```diff
@@ -225,6 +245,8 @@ try {
      * https://www.linkedin.com/in/alan-turing
      * en.linkedin.com/in/alan-turing
      * linkedin.com/in/alan-turing
+     * https://www.linkedin.com/company/linkedin/
+     * journalisttraining@linkedin.com
```

Comment:
do we really want this to be detected?

Reply:
I think we shouldn't add `@linkedin.com` emails, as they seem only for LinkedIn people/departments.

Reply:
Ok, I'll exclude the emails then